src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,154 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head = null;
+    private Node<T> tail = null;
+
     @Override
     public void add(T value) {
+        Node<T> node;
+        node = new Node<>(null, value,null);
```

Comment:
There is no need to declare `node` separately on line 12 and initialize it on line 13. You can do it in one line.

Reply:
fix comment